website/docs/getting-started.md

```diff
@@ -66,6 +67,7 @@ The following configuration options are supported and are all optional.
 | Option | Default | Description |
 | --- | --- | --- |
 | ocrImagesPath | `{project-root}/.tmp` | The folder where the OCR-results are stored |
+| ocrLanguage | `eng` | The language that Tesseract will recognize
```

Comment:
Maybe link to the location where the languages are so people know what they can use

Reply:
👍🏻